go/internal/feast/featurestore.go

```diff
@@ -20,17 +26,24 @@ type FeatureStore struct {
 	onlineStore OnlineStore
 }
 
-type entityKeyRow struct {
-	entityKey  *types.EntityKey
-	rowIndices []int
-}
-
-// A Features struct specifies a list of features to be retrieved from the online store. These features
+// A Features struct specifies a list of Features to be retrieved from the online store. These Features
 // can be specified either as a list of string feature references or as a feature service. String
 // feature references must have format "feature_view:feature", e.g. "customer_fv:daily_transactions".
 type Features struct {
-	features       []string
-	featureService *FeatureService
+	Features       []string
```

Comment:
Should this be called `FeatureRefStrings` or something to be more explicit?

Reply:
how about just `FeatureRefs`?